### PR TITLE
Use WIKIBASE_PORT for the wait-for-it

### PR DESCRIPTION
**Pull request by LeMyst**
wait-for-it accept http/https as PORT

### CODE REVIEW

File: Docker/build/WDQS/runUpdate.sh [hunk 1]
```diff
@@ -4,6 +4,6 @@
 cd /wdqs || exit
 
 # TODO env vars for entity namespaces, scheme and other settings
-/wait-for-it.sh "$WIKIBASE_HOST:80" -t 300 -- \
+/wait-for-it.sh "$WIKIBASE_HOST:$WIKIBASE_SCHEME" -t 300 -- \
```
Comment (toban) [inline Docker/build/WDQS/runUpdate.sh:7]: This is not making alot of sense to me, do you mean we should add WIKIBASE_PORT and use that here?

Comment (LeMyst) [reply Docker/build/WDQS/runUpdate.sh]: Yeah, I missed the existence of variable WIKIBASE_PORT, it's a better fix.

File: Docker/build/WDQS/runUpdate.sh [hunk 2]
```diff
@@ -4,6 +4,6 @@
 cd /wdqs || exit
 
 # TODO env vars for entity namespaces, scheme and other settings
-/wait-for-it.sh "$WIKIBASE_HOST:80" -t 300 -- \
+/wait-for-it.sh "$WIKIBASE_HOST:$WIKIBASE_PORT" -t 300 -- \
 /wait-for-it.sh "$WDQS_HOST:$WDQS_PORT" -t 300 -- \
-./runUpdate.sh -h http://"$WDQS_HOST":"$WDQS_PORT" -- --wikibaseUrl "$WIKIBASE_SCHEME"://"$WIKIBASE_HOST" --conceptUri "$WIKIBASE_SCHEME"://"$WIKIBASE_HOST" --entityNamespaces "$WDQS_ENTITY_NAMESPACES"
+./runUpdate.sh -h http://"$WDQS_HOST":"$WDQS_PORT" -- --wikibaseUrl "$WIKIBASE_SCHEME"://"$WIKIBASE_HOST":"$WIKIBASE_PORT" --conceptUri "$WIKIBASE_SCHEME"://"$WIKIBASE_HOST" --entityNamespaces "$WDQS_ENTITY_NAMESPACES"
```
Comment (LeMyst) [inline Docker/build/WDQS/runUpdate.sh:9]: I've added the $WIKIBASE_PORT to the wikibaseUrl in case we use something else than 80/443

Comment (toban) [reply Docker/build/WDQS/runUpdate.sh:9]: Did you get this working? I think you need to add this to the Dockerfile too

**Comment by addshore** [comment]
CI should now be set up to run on this pull request, but you may need to re push to the branch / rebase on the latest `main`.
Once that is done I can look at reviwing this and getting it in!

**Comment by rti** [comment]
closing this as it is stale for quite a while. feel free to reopen if still relevant

**Pull request closed without merging**